Assets/Develop/Scripts/Player/Controller/PlayerController.cs

```diff
@@ -17,13 +17,15 @@ public class PlayerController : MonoBehaviour
     [SerializeField] [ReadOnly] private string previousStateName;
     [SerializeField] [ReadOnly] private bool hasAirJumped = false;
     [SerializeField] [ReadOnly] private bool hasDashed = false;
-    [SerializeField] [ReadOnly] private bool canUncrouch = false;
     [SerializeField] [ReadOnly] private bool glitchActive = false;
     [SerializeField] [ReadOnly] private int flipGravityScale = 1;
     [SerializeField] [ReadOnly] private WallTrigger wallTrigger = WallTrigger.NONE;
     [SerializeField] private Animator animator;
     [SerializeField] private SkeletonMecanim skeletonMecanim;
 
+    [ReadOnly] private bool canUncrouch = true;
+    [ReadOnly] private bool canUnglitch = true;
```

Comment:
As long as these two fields are `private`, they should be `[SerializeField]`s - in addition to `[ReadOnly]` - to be visible (but not editable) in the Unity inspector.

Reply:
Changed back to `[SerializeField]`